modules/storages/config/locales/en.yml

```diff
@@ -344,6 +344,11 @@ en:
       success:
         subtitle: You are being redirected
         title: Integration setup completed
+      timeout:
+        link_text: health status of the file storage setup
+        subtitle: 'OpenProject could not provide you access to the project folder within the expected period of time. Please, try once again.<br><br> If that problem persists please contact your OpenProject administrator to check the %{storages_health_link}.
+
```

Comment:
🟡 did an empty line sneak in? and a final line break?

Reply:
It was a rake task magic thing :eyes: 
Nice catch, though. :bow: